packages/commonwealth/server/controllers/server_communities_methods/create_chain_node.ts

```diff
@@ -32,7 +40,7 @@ export async function __createChainNode(
   }
 
   const chainNode = await this.models.ChainNode.findOne({
-    where: { url },
+    where: { [Op.or]: { url, eth_chain_id } },
```

Comment:
This breaks cosmos upgrades. Maybe this block can be just for EVM?

Reply:
Fixed this issue